app/controllers/admin/merchants_controller.rb

```diff
@@ -1,5 +1,47 @@
 class Admin::MerchantsController < ApplicationController
   def index
-    
+    @top_5_merchants = Merchant.top_5_merchants_by_total_revenue
```

Comment:
Wow, this is interesting to see in the controller.  Would love to hear about this functionality.